src/loaders/gtdb/gtdb_genome_attribs_loader.py

```diff
@@ -107,53 +84,55 @@ def _row_to_doc(row, kbase_collection, load_version):
     docs = df.apply(_row_to_doc, args=(kbase_collection, load_version), axis=1).to_list()
 
     return docs
 
 
 def main():
-    if not all([header in EXIST_FEATURES.union(NON_EXIST_FEATURES) for header in HEADER_MAPPER.keys()]):
+    if not all([header in loader_common_names.SELECTED_FEATURES for header in HEADER_MAPPER.keys()]):
         raise ValueError('Please make sure HEADER_MAPPER keys are all included in the FEATURES')
 
     parser = argparse.ArgumentParser(
         description='PROTOTYPE - Prepare GTDB genome statistics data in JSON format for arango import.')
+    required = parser.add_argument_group('required named arguments')
+    optional = parser.add_argument_group('optional arguments')
 
     # Required positional argument
-    parser.add_argument('load_files', type=argparse.FileType('r'), nargs='+',
-                        help='GTDB genome metadata files')
+    required.add_argument('load_files', type=argparse.FileType('r'), nargs='+',
+                          help='GTDB genome metadata files (e.g. ar53_metadata_r207.tsv)')
 
     # Required flag argument
-    parser.add_argument(
-        '--load_version', required=True, type=str, nargs=1, help='KBase load version')
+    required.add_argument(f'--{loader_common_names.LOAD_VER_ARG_NAME}', required=True, type=str,
+                          help=loader_common_names.LOAD_VER_DESCR)
 
     # Optional argument
-    parser.add_argument('--kbase_collection', type=str, default='gtdb',
-                        help='kbase collection identifier name (default: gtdb)')
-    parser.add_argument("-o", "--output", type=argparse.FileType('w'), default='gtdb_genome_attribs.json',
-                        help="output JSON file path (default: gtdb_genome_attribs.json")
+    optional.add_argument(f'--{loader_common_names.KBASE_COLLECTION_ARG_NAME}', type=str,
+                          default=names.DEFAULT_KBASE_COLL_NAME,
+                          help=loader_common_names.KBASE_COLLECTION_DESCR)
+    optional.add_argument("-o", "--output", type=argparse.FileType('w'),
+                          default=loader_common_names.GTDB_GENOME_ATTR_FILE,
+                          help=f"output JSON file path (default: {loader_common_names.GTDB_GENOME_ATTR_FILE}")
 
     args = parser.parse_args()
-    load_files, load_version, kbase_collection = args.load_files, args.load_version[0], args.kbase_collection
+    load_files, load_version, kbase_collection = (args.load_files,
+                                                  getattr(args, loader_common_names.LOAD_VER_ARG_NAME),
+                                                  getattr(args, loader_common_names.KBASE_COLLECTION_ARG_NAME))
 
     print('start parsing input files')
-    df = _parse_from_metadata_file(load_files, EXIST_FEATURES, HELPER_FEATURES)
-    df = _compute_attribs(df, NON_EXIST_FEATURES)
-    df.drop(columns=HELPER_FEATURES - EXIST_FEATURES - NON_EXIST_FEATURES,
-            inplace=True)  # drop features added for computation
```

Comment:
So is the ability to define custom functions being dropped?

Reply:
Do you think it's still useful? I feel we will import additional computed features only from running other tools. We will not compute based on this metadata file.

Reply:
:+1: